opencodelists/settings.py

```diff
@@ -234,3 +234,15 @@
     messages.WARNING: "alert-warning",
     messages.ERROR: "alert-danger",
 }
+
+
+# DRF
+
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework.authentication.TokenAuthentication",
+    ],
+    "DEFAULT_PERMISSION_CLASSES": [
+        "rest_framework.permissions.IsAuthenticated",
+    ],
+}
```

Comment:
As there's only one endpoint currently, you could keep this with the view?

Reply:
TIL.  Won't change that now though.